### PR TITLE
Release notes for 6.8.21

### PR DESCRIPTION
**Pull request by logstashmachine**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -52,6 +53,33 @@ This section summarizes the changes in the following releases:
...
+
+=== Logstash Commits between 6.8 and 6.8.20
+
+Computed with "git log --pretty=format:'%h -%d %s (%cr) <%an>' --abbrev-commit --date=relative v6.8.20..6.8"
+
+d55b317 - (HEAD -> 6.8, origin/6.8) Update log4j dependencies (#13500) (2 days ago) <Rob Bavey>
+e8fe311 - Backport PR#12626 to 6.8: use correct headers api for redirects in plugin manager http client (#12626) (#13411) (4 weeks ago) <Rob Bavey>
+afab7d9 - Backport PR #13369 to 6.8. Add java version parameterization to Logstash builds (#13384) (5 weeks ago) <Rob Bavey>
+1285fb6 - Versionbump to 6.8.21 (#13315) (8 weeks ago) <Rob Bavey>
+9d55b4b - Release notes for 6.8.20 (#13305) (9 weeks ago) <Logstash Machine>
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:74]:
```suggestion
```

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -52,6 +53,33 @@ This section summarizes the changes in the following releases:
...
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-21]]
+=== Logstash 6.8.21 Release Notes
+
+---------- DELETE FROM HERE ------------
+=== Logstash Pull Requests with label v6.8.21
+
+* Backport PR #13369 to 6.8. Add java version parameterization to Logst… https://github.com/elastic/logstash/pull/13384[#13384]
+* Backport PR#12626 to 6.8: use correct headers api for redirects in pl… https://github.com/elastic/logstash/pull/13411[#13411]
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:63]:
```suggestion
* Add java version parameterization to Logstash builds https://github.com/elastic/logstash/pull/13384[#13384]
* Use correct headers api for redirects in plugin manager http client https://github.com/elastic/logstash/pull/13411[#13411]
```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -52,6 +53,33 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-21]]
+=== Logstash 6.8.21 Release Notes
+
+---------- DELETE FROM HERE ------------
+=== Logstash Pull Requests with label v6.8.21
+
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:61]:
```suggestion
```

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:74]:
```suggestion
```

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -52,6 +53,21 @@ This section summarizes the changes in the following releases:
...
+* Update log4j dependencies https://github.com/elastic/logstash/pull/13500[#13500]
+
+
+=== Logstash Plugin Release Changelogs ===
+Computed from "git diff v6.8.20..6.8 *.release"
+Changed plugin versions:
+---------- DELETE UP TO HERE ------------
+
+==== Plugins
+
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:70]:
```suggestion
```

File: docs/static/releasenotes.asciidoc [hunk 5]
```diff
@@ -52,6 +53,19 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-21]]
+=== Logstash 6.8.21 Release Notes
+
+[[notable-6-8-21]]
+==== Performance improvements and notable issues fixed
+
+* Add java version parameterization to Logstash builds https://github.com/elastic/logstash/pull/13384[#13384]
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc:62]: This commit is an 'internal' one, only really relevant to CI, should we mark it as such, or skip it entirely from release notes?

Comment (jsvd) [inline docs/static/releasenotes.asciidoc:62]:
```suggestion
```

Comment (jsvd) [reply docs/static/releasenotes.asciidoc]: Removed :+1:

**Review by karenzone — APPROVED** [review]
Builds cleanly, renders as expected, and all links resolve correctly.  LGTM

**Pull request merged**